.github/ISSUE_TEMPLATE/02-feature.yml

```diff
@@ -0,0 +1,52 @@
+name: 🌟 Feature Request
+description: Suggest a feature, minor change, or enhancement to LittleHorse.
+type: "feature"
+body:
+  - type: textarea
+    id: background
+    attributes:
+      label: Background
+      description: Describe the background of this change. What inspired this enhancement?
+  - type: textarea
+    id: proposal
+    attributes:
+      label: Proposal
+      description: Describe your proposed feature, minor change, or enhancement
+    validations:
+      required: true
+  - type: textarea
+    id: acceptance-criteria
+    attributes:
+      label: Acceptance Criteria
+      description: Anything that makes this issue complete.
+    validations:
+      required: true
+  - type: textarea
+    id: scope
+    attributes:
+      label: Out of Scope
+      description: Allows to define what is within the scope of the feature request and what's not.
+  - type: checkboxes
```

Comment:
Can we request that they use labels rather than checkboxes which show up as tasks?